lib/apiservers/engine/proxy/container_proxy.go

```diff
@@ -370,7 +395,9 @@ func (c *ContainerProxy) AddInteractionToContainer(ctx context.Context, handle s
 
 // BindInteraction enables interaction capabilities
 func (c *ContainerProxy) BindInteraction(ctx context.Context, handle string, name string, id string) (string, error) {
-	defer trace.End(trace.Begin(handle))
+	op := trace.FromContext(ctx, "BindInteraction: %s", handle)
+	defer trace.End(trace.Begin(handle, op))
+	// opID := op.ID()
```

Comment:
Are this and the one below that's also commented out are simply incomplete changes?